include/aikido/planner/vectorfield/VectorFieldPlanner.hpp

```diff
@@ -63,31 +93,81 @@ std::unique_ptr<aikido::trajectory::Spline> planPathByVectorField(
 /// \param[in] _constraint Trajectory-wide constraint that must be satisfied
 /// \param[in] _direction Direction of moving the end-effector
 /// \param[in] _distance  Distance of moving the end-effector
-/// \param[in] _linearVelocity Linear velocity of moving the end-effector
-/// \param[in] _linearTolerance How a planned trajectory is allowed to
+/// \param[in] _maxDistance Max distance of moving the end-effector
```

Comment:
I'm confused with what this function wants to do. Does this function plan a trajectory moving the EE for a certain distance (> distance & > maxDistance) along the given direction, which is my interpretation according to the logic of `checkPlanningStatus()`.

Reply:
Once the moved distance is larger than `distance`, it will terminate and return a trajectory.
But if the moved distance is larger than `maxDistance`, it violates and will fail.

Reply:
Yes. You are right. I will update the doc strings accordingly :)